internal/controllers/httpsedge_controller.go

```diff
@@ -315,6 +319,40 @@ func (r *HTTPSEdgeReconciler) setEdgeRouteResponseHeaders(ctx context.Context, e
 	return err
 }
 
+func (r *HTTPSEdgeReconciler) setEdgeRouteWebhookVerification(ctx context.Context, edge *ingressv1alpha1.HTTPSEdge, routeID string, webhookValidation *ingressv1alpha1.EndpointWebhookValidation) error {
+	client := r.NgrokClientset.EdgeModules().HTTPS().Routes().WebhookVerification()
+	if webhookValidation == nil {
+		return client.Delete(ctx, &ngrok.EdgeRouteItem{EdgeID: edge.Status.ID, ID: routeID})
+	}
+
+	// TODO: Get the secret from the store
```

Comment:
Discussed offline, but the edge controller won't be accessing the store, but this Client get is fine since it should rely on the controller-runtime manager's cache https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/cluster/cluster.go#L49-L53

Reply:
Ahhh....yes. that is right.

Reply:
Removed in [80ae704](https://github.com/ngrok/kubernetes-ingress-controller/pull/122/commits/80ae7046f0733b6afd98ade63803a40d57ac6345)